cookbook/migration-v1-v2.md

```diff
@@ -78,21 +75,21 @@ The type `PiniaStorePlugin` was renamed to `PiniaPlugin`.
  }
 ```
 
-**Note this change can only be done after upgrading to the latest version of Pinia without deprecations**.
+**Tenga en cuenta que este cambio sólo se puede hacer después de actualizar a la última versión de Pinia sin depreciaciones**.
 
-## `@vue/composition-api` version
+## Versión `@vue/composition-api` {#vue-composition-api-version}
 
-Since pinia now relies on `effectScope()`, you must use at least the version `1.1.0` of `@vue/composition-api`:
+Como pinia depende ahora de `effectScope()`, debes usar al menos la versión `1.1.0` de `@vue/composition-api`:
 
 ```shell
 npm i @vue/composition-api@latest
-# or with yarn
+# o con yarn
 yarn add @vue/composition-api@latest
 ```
 
-## webpack 4 support
+## soporte con webpack 4 {#webpack-4-support}
```

Comment:
Since is a title `soporte` must begin with a mayus

Reply:
done